cli/src/commands/start-prover/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Command, Flags } from "@oclif/core";
+import { CustomLoader } from "../../utils/index";
+import { startProver } from "../../utils/process";
+
+class StartProver extends Command {
+  static description = "Start gnark prover";
+
+  protected finally(_: Error | undefined): Promise<any> {
+    process.exit();
+  }
+
+  static flags = {
+    "prove-compressed-accounts": Flags.boolean({
+      description: "Enable proving of compressed accounts.",
+      default: false,
+      char: "c",
+    }),
+    "prove-new-addresses": Flags.boolean({
```

Comment:
can we by default start with all proving keys and add flags to skip provingkeys?
I feel like that way it will be easier to use.
``skip-prove-new-addresses``